internal/service/repository.go

```diff
@@ -64,3 +82,19 @@ func (r *Repository) Rules() (*Rules, error) {
 
 	return &rules, nil
 }
+
+// RemoteConfiguration returns a default remote configuration for v2 endpoint
+func (r *Repository) RemoteConfiguration() (*RemoteConfiguration, error) {
+	filepath := "config_default.json" // TODO: Make this depend on OCP version
```

Comment:
when is this TODO going to be done?

Reply:
After we talk about how the mapping should be done (semantic versions/regexes) with Jan Holeček (after he returns from PTO).

Reply:
please create a task so that we don't forget!

Reply:
It's still part of the initial [CCXDEV-12785](https://issues.redhat.com/browse/CCXDEV-12785), that has been postponed to next sprint (even though it's almost done, except for this thing). I'll leave a comment there.